project/Dependencies.scala

```diff
@@ -361,11 +359,10 @@ object Dependencies {
     dropwizardMetricsCore,
     "com.typesafe.netty" % "netty-reactive-streams" % NettyReactiveStreamsVersion,
     "io.netty" % "netty-codec-http" % NettyVersion,
+    scalaTest % Test,
```

Comment:
This wasn't an override, just a normal dependency. Moved it up to make the more clear.